libwallet/src/slate_versions/v4.rs

```diff
@@ -34,7 +34,8 @@
 //! *  The `payment_proof` struct is renamed to `proof`
 //! * The feat_args struct is added, which may be populated for non-Plain kernels
 //! * `proof` may be omitted from the slate if it is None (null),
-//! * `offset` is added, which may be optionally included during S3 and I3 to ensure the transaction can be re-built entirely from the slate information. Used for delayed transaction posting.
+//! * `off` (offset) is added, and will be modified by every participant in the transaction with a random
+//! value - the value of their inputs
```

Comment:
not the value of their inputs, but their blinding factor.

Reply:
corrected